tools/packaging/static-build/ovmf/build-ovmf.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+#
+# Copyright (c) 2022 IBM
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+source "${script_dir}/../../scripts/lib.sh"
+
+# disabling set -u because scripts attempt to expand undefined variables
+set +u
+ovmf_repo="${ovmf_repo:-}"
+ovmf_dir="edk2"
+ovmf_version="${ovmf_version:-}"
+kata_version="${kata_version:-}"
+DESTDIR="${DESTDIR:-../../destdir}"
+PREFIX="${PREFIX:-/opt/kata}"
+ovmf_build="${ovmf_build:-sev}"
+architecture="X64"
+toolchain="GCC5"
+build_target="RELEASE"
+
+if [ -z "$ovmf_repo" ]; then
+       info "Get ovmf information from runtime versions.yaml"
+       ovmf_repo=$(get_from_kata_deps "externals.ovmf.url" "${kata_version}")
+fi
+[ -n "$ovmf_repo" ] || die "failed to get ovmf repo"
+
+if [ "${ovmf_build}" == "sev" ]; then
+       [ -n "$ovmf_version" ] || ovmf_version=$(get_from_kata_deps "externals.ovmf.sev.version" "${kata_version}")
+       [ -n "$ovmf_package" ] || ovmf_package=$(get_from_kata_deps "externals.ovmf.sev.package" "${kata_version}")
+       [ -n "$package_output_dir" ] || package_output_dir=$(get_from_kata_deps "externals.ovmf.sev.package_output_dir" "${kata_version}")
+elif [ "${ovmf_build}" == "x64" ]; then
+       [ -n "$ovmf_version" ] || ovmf_version=$(get_from_kata_deps "externals.ovmf.x64.version" "${kata_version}")
+       [ -n "$ovmf_package" ] || ovmf_package=$(get_from_kata_deps "externals.ovmf.x64.package" "${kata_version}")
+       [ -n "$package_output_dir" ] || package_output_dir=$(get_from_kata_deps "externals.ovmf.x64.package_output_dir" "${kata_version}")
+
+fi
```

Comment:
I'd prefer to have those being introduced in two different commits.
The first one adding support for building OVMF, and a second one adding support for building OVMF for SEV.  That would be cleaner.